src/components/TopicList/README.md

```diff
@@ -30,43 +29,42 @@ class TopicListExample extends React.Component {
     );
 
     this.setState({
-      layerService: new LayerService(layers),
-    })
+      layers,
+    });
   }
 
   onTopicClick(topic) {
-    const { topics, layerService } = this.state;
-    layerService.layers.forEach(layer => layer.setVisible(false));
+    const { topics, layers } = this.state;
+    layers.forEach(layer => layer.setVisible(false));
 
     topics.map(t => {
       t.visible = (t.id === topic.id);
       t.expanded = true;
       return t;
     });
 
-    const layers = ConfigReader.readConfig(
+    const newLayers = ConfigReader.readConfig(
       this.map,
       ConfigReader.getVisibleTopic(topics).children,
     );
 
     this.setState({
```

Comment:
Just remove this setState use.

Reply:
I created a new branch and pull request (https://github.com/geops/react-spatial/pull/57) as there were too many changes in LayerTree which I reset. And I didn't want to squash all commits as there are other, independent bug fixes which I had fixed while working on the old branch.